Dockerfile does not build, missing ABOUT.md

Type: Bug Report

- [x] Try to follow the update procedure described in the README and try again before opening this issue.
- [x] Please check the [F.A.Q.](https://github.com/Flood-UI/flood/wiki/F.A.Q.).
- [x] Please check the [Troubleshooting](https://github.com/Flood-UI/flood/wiki/Troubleshooting) wiki section.

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* Version used:
  + commit 61b63e9b549b4ae92796b5595f583e4cff388a13
* Environment name and version:
  + Node.js version v10.1.0
    - Docker image node 10.1-alpine
  + npm version 5.6.0
  + Web browser N/A
* Operating System and version:
  + Debian 9
  + Docker version 18.09.5, build e8ff056dbc 

## Summary
Docker image fails to build in Step 12, due to required file `ABOUT.md` not being a part of the COPY steps and not being present for the build:

```
Step 12/21 : RUN npm run build &&     npm prune --production
 ---> Running in cdd2bdc9e83f

> flood@1.0.0 build /usr/src/app
> node client/scripts/build.js

(node:17) ExperimentalWarning: The fs.promises API is experimental
Creating an optimized production build...
Failed to compile.

Module not found: Error: Can't resolve '../../../../../../ABOUT.md' in '/usr/src/app/client/src/javascript/components/modals/settings-modal'
```

The include is in file `client/src/javascript/components/modals/settings-modal/AboutTab.js`

## Possible Solution

Include ABOUT.md in build.

## Steps to Reproduce
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
1. `docker build .`

--- a/Dockerfile
+++ b/Dockerfile
@@ -19,6 +19,7 @@ RUN apk add --no-cache --virtual=build-dependencies \
 COPY client ./client
 COPY shared ./shared
 COPY config.docker.js ./config.js
+COPY ABOUT.md ./ABOUT.md
 RUN npm run build && \
     npm prune --production
 COPY server ./server

